README.md

```diff
@@ -1,2 +1,42 @@
 Memoization decorator for python 
-Store function results on disk. Load if call to the function was previously stored. 
+=== 
+Add @Memoized before function definition for automatic caching. 
+
+On the first call to the function the decorator runs it with the provided parameters. Then it stores the returned result in a file. In subsequent calls to the function with the same parameters, the decorator returns the results from the cache file without calling the function. The decorator matches the function text and the parameters, so if the function or the parameters change the function will be run again. 
+
+    from memoize.memoized import Memoized
+    import time
+
+    @Memoized(cacheDir = "/home/username/tmp") # cacheDir defaults to /tmp in Linux 
+    def fibonacci(n):
+        if n <= 2: return 1
```

Comment:
Let's use a shorter, idiomatic implementation - see [here](https://stackoverflow.com/a/15047141/51197). For example, we can do `a, b, k = 1, 1, 2`.

Reply:
done.